### DISCUSSION TITLE
2주차 스터디 회고

### DISCUSSION THREAD

**Discussion by alanhakhyeonsong** [Next-Level-Study/effective-java-study#7] [스터디 회고]
한 줄 회고와 건의사항을 말씀해주세요:)

**Comment by JeongJaew0n** [comment]
핵심 내용 정도만 알아두고 넘어 갈려 했는데 자세히 보니 고민해볼 내용이 정말 많네요. 다음 주에는 조금 더 고민하고 깊이 있게 공부해봐야 겠습니다.

**Comment by alanhakhyeonsong** [comment]
그냥 정리하고 이야기하고 끝이 아니라 우리는 개발자니까 코드로 확인해보는 습관을 더 들여봐야겠습니다.

**Comment by Kurmarkpzgr** [comment]
무심코 넘어간 부분이 상당히 있었는데, 한 번 씩 다시 짚어보는 점이 참 뜻깉은 시간이 되었던거 같습니다.

**Comment by opgumo3** [comment]
그냥 그러겠지~ 하고 넘어간 부분들이 있었는데 좋지 않은 행동이었던 것 같습니다.
책 내용 외로도 찾아보고 해야 할 것 같습니다.

**Comment by SuJeong68** [comment]
읽는 것으로만으로도 어려워 넘어가버린 부분들이 많다는 것을 알 수 있었습니다. 더 열심히 공부하고 생각해봐야겠습니다 😂

**Comment by narangd0** [comment]
평소에 고민해보지 않았던 원칙들을 알아볼 수 있는 내용이어서 좋았습니다..
깊게 들여다보는 게 어렵지만... 앞으로 조금씩 더 노력하도록 하겠습니다!

**Reply by narangd0** [reply]
건의! 3주? 4주? 마다 지난 내용들 복습 + 부족했던 내용 추가 정리 + 심화 내용에 대해 논의할 수 있는 시간이 있으면 좋겠습니다~!

**Comment by sdh46** [comment]
완전히 새로운 내용들보다는 조금씩 아는 부분도 있었지만 그 중에서 깊이 있고 명확하게 알고 쓰는 것은 또 다르다고 느꼈습니다.
